packages/protocol/contracts/governance/ReleaseGoldInstance.sol

```diff
@@ -0,0 +1,579 @@
+   * @notice A convenience wrapper setter for the name, dataEncryptionKey
+   *         and wallet address for an account.
+   * @param name A string to set as the name of the account.
+   * @param dataEncryptionKey secp256k1 public key for data encryption. Preferably compressed.
+   * @param walletAddress The wallet address to set for the account.
+   */
+  function setAccount(string calldata name, bytes calldata dataEncryptionKey, address walletAddress)
+    external
+    onlyWhenInProperState
+  {
+    getAccounts().setAccount(name, dataEncryptionKey, walletAddress);
+  }
+
+  /**
+   * @notice A wrapper setter function for creating an account.
+   */
+  function createAccount() external onlyWhenInProperState {
+    require(canVote, "Account cannot be created if voting is disabled");
+    require(getAccounts().createAccount(), "Account creation failed");
+  }
+
+  /**
+   * @notice A wrapper setter function for the name of an account.
+   * @param name A string to set as the name of the account.
+   */
+  function setAccountName(string calldata name) external onlyWhenInProperState {
+    getAccounts().setName(name);
+  }
+
+  /**
+   * @notice A wrapper setter function for the wallet address of an account.
+   * @param walletAddress The wallet address to set for the account.
+   */
+  function setAccountWalletAddress(address walletAddress) external onlyWhenInProperState {
+    getAccounts().setWalletAddress(walletAddress);
+  }
+
+  /**
+   * @notice A wrapper setter function for the for the data encryption key
+   *         and version of an account.
+   * @param dataEncryptionKey Secp256k1 public key for data encryption.
+   *                          Preferably compressed.
+   */
+  function setAccountDataEncryptionKey(bytes calldata dataEncryptionKey)
+    external
+    onlyWhenInProperState
+  {
+    getAccounts().setAccountDataEncryptionKey(dataEncryptionKey);
+  }
+
+  /**
+   * @notice A wrapper setter function for the metadata of an account.
+   * @param metadataURL The URL to access the metadata..
+   */
+  function setAccountMetadataURL(string calldata metadataURL) external onlyWhenInProperState {
+    getAccounts().setMetadataURL(metadataURL);
+  }
+
+  /**
+   * @notice Increments the number of total and pending votes for `group`.
+   * @param group The validator group to vote for.
+   * @param value The amount of gold to use to vote.
+   * @param lesser The group receiving fewer votes than `group`, or 0 if `group` has the
+   *               fewest votes of any validator group.
+   * @param greater The group receiving more votes than `group`, or 0 if `group` has the
+   *                most votes of any validator group.
+   * @dev Fails if `group` is empty or not a validator group.
+   */
+  function vote(address group, uint256 value, address lesser, address greater)
+    external
+    nonReentrant
+    onlyWhenInProperState
+  {
+    require(getElection().vote(group, value, lesser, greater), "Voting for a group failed");
```

Comment:
Do the other wrapped functions return booleans? If so, for cleanliness, let's be consistent in requiring them (or not)

Reply:
All functions that don't `require` don't return bools. However, all of the bools returned are useless (imo) since they just return true. I'll remove the requires.